python/pyspark/sql/dataframe.pyi

```diff
@@ -86,7 +87,7 @@ class DataFrame(PandasMapOpsMixin, PandasConversionMixin):
     def withWatermark(
         self, eventTime: ColumnOrName, delayThreshold: str
     ) -> DataFrame: ...
-    def hint(self, name: str, *parameters: Any) -> DataFrame: ...
+    def hint(self, name: str,  *parameters: Union[PrimitiveType, List[PrimitiveType]]) -> DataFrame: ...
```

Comment:
Nitpick: There seems to be an additional whitespace before `*parameters`.

Reply:
Thank you @zero323.